main.py

```diff
@@ -16,48 +16,76 @@
 
 
 def handle_command(command, channel, msg, usernm):
-     """
-         Receives commands directed at the bot and determines if they
-         are valid commands. If so, then acts on the commands. If not,
-         returns back what it needs for clarification.
-     """
-     response = ""
-     key=str(msg[0]["text"]).split(' ')[1][1:]
-     totText=str(msg[0]["text"])
-     totText=totText[totText.find(' '):]
-     totText=totText[6:]
-     totText=totText[totText.find(' '):]
-     with open("data.json") as json_file:
-         data = json.load(json_file)
-         flag=1
-         try:
-           handles=data[key]
-         except:
-            response = "Not sure what you mean. Use the */freshers* or */seniors* command with text separated by a single space to notify them.\nI\'ll ping you at 3 in case you are awake! :smile:"
-            flag=0
-         if flag==1:
-           for i in handles:
-              response+="<@"+str(i)+"> "
-           response+= "\nNotification for *"+str(key)+"* from <@"+usernm+">: "+totText
-     print "\nResponse: -\n"+response
-     slack_client.api_call("chat.postMessage", channel=channel,
-                           text=response, as_user=True)
+	 """
+		 Receives commands directed at the bot and determines if they
+		 are valid commands. If so, then acts on the commands. If not,
+		 returns back what it needs for clarification.
+	 """
+	keys = []
+	response = ""
+	msg = (msg.replace("@bhattu","")).strip()
+	comp = msg.split()
+	seniors_msg = ""
+	freshers_msg = ""
```

Comment:
@dibyadas Don't do this ^ ...
In this case, the code is restricted to only freshers and seniors...we should be able to add more commands by simply modifying the JSON, not the code...